frontend/src/views/Dashboard.vue

```diff
@@ -69,5 +70,20 @@ export default class Dashboard extends Vue {
     }
     return "Home";
   }
+
+  customBreadcrumb(paths: any) {
+    if(paths && paths[1] == "reporting" && paths[2] == "content") {
```

Comment:
Please implement this in a more systematic way. Dashboard should not know about any particular route, like Content Reporting. It is improper to handle all the unusual cases here. Think about creating a route meta-property with a callback for this.